zip-xxxx.rst

```diff
@@ -0,0 +1,290 @@
+::
+
+  ZIP: 230
+  Title: Version 6 Transaction Format
+  Owners: Daira Emma Hopwood <daira@electriccoin.co>
+          Jack Grigg <jack@electriccoin.co>
+          Sean Bowe <sean@electriccoin.co>
+          Kris Nuttycombe <kris@electriccoin.co>
+          Greg Pfeil <greg@electriccoin.co>
+          Deirdre Connolly <deirdre@zfnd.org>
+  Credits: Ying Tong Lai
+  Status: DRAFT
+  Category: Consensus
+  Created: 2023-04-18
+  License: MIT
+  Discussions-To: <https://github.com/zcash/zips/issues/686>
+
+
+Terminology
+===========
+
+The key words "MUST" and "MAY" in this document are to be interpreted as described in
+RFC 2119. [#RFC2119]_
+
+The character § is used when referring to sections of the Zcash Protocol Specification
+[#protocol]_.
+
+
+Abstract
+========
+
+This proposal defines an update to the Zcash peer-to-peer transaction format to
+include support for data elements required to support Zcash Shielded Assets,
+initially specified to support the Orchard shielded pool [#protocol]_.  The
+new transaction format defines well-bounded regions of the serialized form to
+serve each of the existing pools of funds, and adds and describes a new elements
+containing ZSA-specific elements.
+
+This ZIP also depends upon and defines modifications to the computation of the values
+**TxId Digest**, **Signature Digest**, and **Authorizing Data Commitment** defined by ZIP
+244 [#zip-0244]_.
+
+
+Motivation
+==========
+
+The new Orchard shielded pool requires serialized data elements that are distinct from
+any previous Zcash transaction. In addition, with the activation of ZIP 244, the
+serialized transaction format will no longer be consensus-critical. It makes sense at this
+point to define a format that can readily accommodate future extension in a systematic
+fashion, where elements required for support for a given pool are kept separate.
```

Comment:
1. ZIP-244 has already been activated
2. Reduce redundant wording

```suggestion
any previous Zcash transaction. Since ZIP 244 was activated in NU5, the
v5 and later serialized transaction formats are not consensus-critical. So this ZIP defines
a format that can easily accommodate future extensions, where elements or a given pool
are kept separate.
```